test/common/upstream/utility.h

```diff
@@ -124,7 +137,7 @@ makeLocalityWeights(std::initializer_list<uint32_t> locality_weights) {
 }
 
 inline envoy::config::core::v3::HealthCheck
-parseHealthCheckFromV2Yaml(const std::string& yaml_string) {
+parseHealthCheckFromV3Yaml(const std::string& yaml_string) {
   envoy::config::core::v3::HealthCheck health_check;
   TestUtility::loadFromYamlAndValidate(yaml_string, health_check);
```

Comment:
Do you need to force v3 here so we don't get any accidental v2 configs going through?

Reply:
@htuch yes

Reply:
This one is still missing forcing v3.. can you ping me when this is ready for another pass? Thanks.
/wait